lib/repl.js

```diff
@@ -964,6 +964,9 @@ function complete(line, callback) {
   // REPL commands (e.g. ".break").
   var filter;
   var match = null;
+
+  var functionCallRE = /\b(\w+)\([^)]*$/;
```

Comment:
I think there can be arbitrary whitespace before the opening `(`?

Reply:
Yes. I will add test cases and go through the code again to make sure its production ready. meanwhile i just wanted to submit the PR to get feedback for the general direction of the feature rather than the code details.

Reply:
Misses `$`, but otherwise seems OK. `%` may also be needed for natives support.

Reply:
nit: this can be `const`